airflow/api_connexion/schemas/dag_schema.py

```diff
@@ -113,6 +113,7 @@ class DAGDetailSchema(DAGSchema):
     template_searchpath = fields.String(dump_only=True)
     render_template_as_native_obj = fields.Boolean(dump_only=True)
     last_loaded = fields.DateTime(dump_only=True, data_key="last_parsed")
+    call_on_kill_on_dagrun_timeout = fields.Boolean(dump_only=True)
```

Comment:
What would be the scenario where dagrun/task timeout and you wouldn't want to execute the `on_kill`?